scripts/send-tasks.gmp

```diff
@@ -176,18 +180,15 @@ def parse_send_xml_tree(gmp, xml_tree):
 
 def main():
     # pylint: disable=undefined-variable
-
     check_args(args)
+    xml_doc = args.script[1]
 
-    xml_doc = args.script[2]
-
-    print('\nSending tasks...')
+    print('\nSending task(s)...')
 
     xml_tree = create_xml_tree(xml_doc)
     parse_send_xml_tree(gmp, xml_tree)
 
     print('\n  Task(s) sent!\n')
 
-
-if __name__ == '__gmp__':
+if __name__ == 'gmp.clients.gvm_pyshell':
```

Comment:
This line must be kept. See the [CHANGELOG](https://github.com/greenbone/gvm-tools/blob/master/CHANGELOG.md#scripts) for an explanation